web/src/views/Dashboard.tsx

```diff
@@ -112,19 +101,45 @@ const Dashboard: React.FC<IDashboardProps> = (props) => {
       })
   }, 1000 * 30);
 
+  const downloadCSV = () => {
+    openBackdropCB();
+    try {
+      const clonedProfile = Array.from(profile);
+
+      clonedProfile.filter((item: any) => item.provisioned === true);
+      clonedProfile.forEach((item: any) => {
+        delete item.criticalSystem;
```

Comment:
I generally favour the spread operator to get what I want rather than removing to create what I want. The reason is if some other value works its way into `clonedProfile` it will appear in the output (CSV) and maybe you want that, maybe you don't. Using the spread operator you always get what you want.